infrastructure/kubernetes/modules/env/main.tf

```diff
@@ -146,9 +146,13 @@ module "k8s_tiler" {
     {
       name = "DEFAULT_COG"
       value = "biomass.tif"
+    },
+    {
+      name = "REQUIRE_AUTH"
+      value = "false"
```

Comment:
I am setting this as false (for now) because:

1. Adding a auth method in FastAPI's swagger apparently is not as trivial as it is in other frameworks
2. Science team will likely be testing this through swagger until we get the right configuration
3. The cog served by the tiler is a generic one for testing purposes